src/control/traction.cpp

```
@@ -67,38 +86,84 @@ inline void applyHardwareControl(int wheelIndex, uint16_t pwmTicks, bool reverse
+      // ✅ NUEVO v2.11.5: Validar canales PWM antes de escribir
+      bool fwdValid = validatePWMChannel(PCA_FRONT_CH_FL_FWD, "FL_FORWARD");
+      bool revValid = validatePWMChannel(PCA_FRONT_CH_FL_REV, "FL_REVERSE");
+      
+      if (fwdValid && revValid) {
+        pcaFront.setPWM(PCA_FRONT_CH_FL_FWD, 0, reverse ? 0 : pwmTicks);
+        pcaFront.setPWM(PCA_FRONT_CH_FL_REV, 0, reverse ? pwmTicks : 0);
+      } else {
+        // Safety: establecer ambos a 0 si algún canal es inválido
+        if (fwdValid) pcaFront.setPWM(PCA_FRONT_CH_FL_FWD, 0, 0);
+        if (revValid) pcaFront.setPWM(PCA_FRONT_CH_FL_REV, 0, 0);
+        Logger::errorf("PWM: Channel pair invalid FL (fwd:%d, rev:%d)", fwdValid, revValid);
```

Clang (14.0.6)

[warning] do not call c-style vararg functions

(cppcoreguidelines-pro-type-vararg)